{"name": "test_broken", "status": "broken", "statusDetails": {"message": "Exception: oops", "trace": "def test_broken():\n>       raise Exception('oops')\nE       Exception: oops\n\ntest_alluredemo.py:23: Exception"}, "start": 1611749448962, "stop": 1611749448962, "uuid": "dd82b303-68c0-44bc-a968-39741a06fc01", "historyId": "fb799eecaee64eb20ac70a0047ddd7ff", "testCaseId": "54327feac6bee8ccd45a88d670bf2886", "fullName": "test_alluredemo#test_broken", "labels": [{"name": "suite", "value": "test_alluredemo"}, {"name": "host", "value": "Naraku15"}, {"name": "thread", "value": "17252-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_alluredemo"}]}